helmfile/values/ingress-nginx.yaml.gotmpl

```diff
@@ -1,12 +1,17 @@
 ## nginx configuration
-## Ref: https://github.com/kubernetes/ingress/blob/master/controllers/nginx/configuration.md
+## Ref: https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/index.md
 ##
+
 controller:
+  name: controller
   image:
-    allowPrivilegeEscalation: true
-    tag: "0.28.0"
-    runAsUser: 101
+    # for backwards compatibility consider setting the full image url via the repository value below
+    # use *either* current default registry/image or repository format or installing chart by providing the values.yaml will fail
+    # repository:
+    tag: "v0.48.1"
+    digest: sha256:e9fb216ace49dfa4a5983b183067e97496e7a8b307d2093f4278cd550c303899
```

Comment:
This as well

Reply:
Ok, but hopefully we will not unintentionally update the image version when we update the chart version.

Reply:
We usually do want to update the image when we update the chart. So it should be ok.

Reply:
ok